.github/workflows/test.yml

```
@@ -119,9 +194,93 @@ jobs:
+      GRAFANA_SERVICE_ACCOUNT_TOKEN: ${{ secrets.GRAFANA_SERVICE_ACCOUNT_TOKEN }}
+      BALANCE_CHANGES: false
+      WARP_CONFIG_CHANGES: false
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha || github.sha }}
+          submodules: recursive
+          fetch-depth: 0
+
+      - name: Check for balance/config changes
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:34: Double quote to prevent globbing and word splitting

(shellcheck)